src/System.Private.CoreLib/src/System/Environment.CoreCLR.cs

```diff
@@ -83,10 +83,12 @@ public static string[] GetCommandLineArgs()
             get;
         }
 
-        public static int ProcessorCount => GetProcessorCount();
+        public static int ProcessorCount => GetProcessorCount(withCpuLimit: false);
+
+        internal static int ProcessorQuota => GetProcessorCount(withCpuLimit: true);
```

Comment:
This is not consumed by anything. We typically try to avoid future proofing - adding dead code that may be useful one day.